js/templates/modals/orderDetail/summaryTab/summary.html

```diff
@@ -8,6 +8,7 @@ <h1 class="inline tx4"><%= ob.polyT('orderDetail.summaryTab.orderNumber', { orde
 </div>
 <hr class="clrBr rowLg" />
 <div class="js-statusProgressBarContainer statusProgressBarContainer"></div>
+<p class="js-paymentFinalizedMsg clrTErr hide" style="margin-top: 10px"><span class="ion-alert-circled padSm"></span><%= ob.polyT('orderDetail.summaryTab.vendorFinalizedPayment') %></p>
```

Comment:
Cut a corner and embedded some css, since this is a temporary message that we'll very likely be refactoring.